include/gridtools/stencil_composition/backend_mc/fused_mss_loop_mc.hpp

```diff
@@ -61,7 +61,7 @@ namespace gridtools {
         class LocalDomainListArray,
         class Grid,
         enable_if_t<_impl::all_mss_kparallel<MssComponents>::value, int> = 0>
-    GT_FORCE_INLINE static void fused_mss_loop(
+    static void fused_mss_loop(
```

Comment:
These force inline _had_ some severe impact many months ago on one stencil and one compiler, but don’t remember which and why. But as a lot has changed since then it’s probably fine to remove if there’s no performance difference…

Reply:
gcc 8 on my machine complains that this function can not be inlined (probably because omp pragmas in it.)